tasks/libs/junit_upload_core.py

```diff
@@ -24,6 +25,8 @@
 JOB_URL_FILE_NAME = "job_url.txt"
 JOB_ENV_FILE_NAME = "job_env.txt"
 TAGS_FILE_NAME = "tags.txt"
+TEST_OUTPUT_FILES = ["test_output.json", "test_output_fast.json"]
+FLAKE_MESSAGES = ["flakytest: this is a known flaky test", "flaky test in CI"]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
We should track only the known and standard marker to push people to adopt it - we might want to always skip marked tests at some point, when we have a job to run only marked tests
```suggestion
FLAKE_MESSAGES = ["flakytest: this is a known flaky test"]
```

Reply:
I'm ok with that, then we should probably [replace this](https://github.com/DataDog/datadog-agent/blob/main/pkg/process/procutil/process_linux_test.go#L1397) by a call to `flake.Mark`, wdyt?

Reply:
Going back to just tracking the `flake.Mark` message